src/code/scopes.js

```diff
@@ -34,33 +34,42 @@
             label: "let decay",
             color: "#7D7D7D",
             markerIcon: scope.static.images.marker_alert_decay
-        }
-        /*, {
+        }, {
+	    name: Wasabee.Constants.MARKER_TYPE_KEY,
+	    label: "get key",
+	    color: "#7D7D7D",
+	    markerIcon: scope.static.images.marker_alert_key
+	}, {
+	    name: Wasabee.Constants.MARKER_TYPE_LINK,
```

Comment:
It looks like this file somehow got messed up. I see a lot of places where name is repeated.